wcbench.sh

```diff
@@ -17,6 +17,9 @@ EX_ERR=1
 # Output verbose debug info (true) or not (anything else)
 VERBOSE=false
 
+# Checking the OS
+OS=`echo $OSTYPE`
```

Comment:
This doesn't work on Fedora. Either use a better check or (recommended) add this to the other var blocks, set it to `fedora` by default (to remain API-backward-compatible) and allow people to switch it to `ubuntu` if they are on Ubuntu systems.